src/date-picker/index.ts

```diff
@@ -33,7 +30,7 @@ export class DatePicker extends KucBase {
   @property({ type: Boolean }) disabled = false;
   @property({ type: Boolean }) requiredIcon = false;
   @property({ type: String }) language = "auto";
-  @property({ type: String }) value = getTodayStringByLocale();
+  @property({ type: String }) value? = getTodayStringByLocale();
```

Comment:
@h000780 
Following the spec, the DatePicker.value only accept the date format `YYYY-MM-DD`.
It will be the same with `_baseDateValue` have been declared.
So, please using `@property({ type: String }) value` for base date value, remove `_baseDateValue`.

The `getTodayStringByLocale()` using for `_inputValue`.
Please declare `private _inputValue` to use.